internal/cmd/kafka/command_acl_list.go

```diff
@@ -25,6 +25,7 @@ func (c *aclCommand) newListCommand() *cobra.Command {
 	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
 	pcmd.AddServiceAccountFlag(cmd, c.AuthenticatedCLICommand)
 	cmd.Flags().String("principal", "", `Principal for this operation, prefixed with "User:".`)
+	cmd.Flags().Bool("all", false, "Include ACLs for principals with integer IDs. These ACLs are associated with deleted principals.")
```

Comment:
```suggestion
	cmd.Flags().Bool("all", false, "Include ACLs for deleted principals with integer IDs.")
```

Reply:
I like this.

It explains what the integer principals mean but stops short of implying that all of the deleted principals will have integer ids. Thanks.